components/chips.json

```diff
@@ -0,0 +1,21 @@
+{
+  "chips": {
+    "comment": "Applied to Chips",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=44750%3A103138",
```

Comment:
let's change the figma link to https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=5028%3A270 to be more concrete